tagging-status/CONTRIBUTING.md

```diff
@@ -29,21 +29,27 @@ Compile the test file and check the output using software such as PAC or VeraPDF
 
 Run `l3build save <class or package name>-01` and `l3build save -e pdftex <class or package name>-01` (for multiple test files, just concatenate the names) in your working directory to create an XML file named `<class or package name>-01.struct.xml` in the `testfiles/<class or package name>/` subdirectory. Check the contents of this file as well. This file shows a combination of tagging, structure and alternative representations which can also help you to check the tagging support of your class or package. This file will also serve as test file against which the tagging status of your class or package is later checked against.  
 
+If you spot XML comments such as `<!-- UTF16 to UTF8 conversion failure -->` in the generated XML file, it is likely that `cm-super` is not installed. These comments appear as there are not Unicode mappings for bitmap fonts which are maybe used because `cm-super` is not installed. 
```

Comment:
which might get used

Reply:
Thanks, added this and also introduced line breaks between semantic breaks. 

Sorry, as the structure of the repository has changed, I decided to rewrite the relevant passage in the file.